tls/s2n_cipher_suites.c

```diff
@@ -153,6 +155,23 @@ const struct s2n_record_algorithm s2n_record_alg_chacha20_poly1305 = {
     .flags = S2N_TLS12_CHACHA_POLY_AEAD_NONCE,
 };
 
+/* TLS 1.3 Record Algorithms */
+const struct s2n_record_algorithm s2n_tls13_record_alg_aes128 = {
+    .cipher = &s2n_tls13_aes128_gcm,
+    .hmac_alg = S2N_HMAC_NONE, /* previously used in 1.2 prf, we do not need this */
+    .flags = S2N_TLS13_IMPLICIT_IV,
```

Comment:
perhaps rename `S2N_TLS13_IMPLICIT_IV` to `S2N_TLS13_RECORD_NONCE` to follow conventions

Reply:
+1 for following conventions